src/inject/inject.js

```diff
@@ -40,6 +44,10 @@ function insertButton() {
         to_match = to_match.concat("yt-uix-tile-link yt-ui-ellipsis yt-ui-ellipsis-2 yt-uix-sessionlink      spf-link ", '"]');
         LOC_HREF = 4;
     }
+    else if (document.location.href == "https://www.youtube.com/feed/channels") {
+        to_match = to_match.concat("yt-uix-tile-link yt-ui-ellipsis yt-ui-ellipsis-2 yt-uix-sessionlink      spf-link ", '"]');
+        LOC_HREF = 5;
```

Comment:
I don't think this works. Have you checked it?

IMO there should be substring check like on line no 35.
https://github.com/Manu-shukla/Play_Next/blob/c4baf927cc5a065c8be96866085ae160f3612b52/src/inject/inject.js#L35

Reply:
But, locally it is working on my system. I think it is due to cookie being saved, I'll check it once.

Reply:
Could you post a screenshot if it works on your laptop, even if it works for your personal channel IMO it won't work for every channel. Kindly check

Reply:
Ok, let me check.